wtpython/__main__.py

```diff
@@ -67,6 +100,9 @@ def main() -> None:
         return
 
     error = ''.join(traceback.format_exception_only(type(exc), exc)).strip()
+    error_lines = error.split('\n')
+    if len(error_lines) > 1:
+        error = error_lines[-1]
```

Comment:
What are these lines for?

Reply:
Ah shit, this should have been a separate commit. Basically, in some cases (I think when you have a syntax error as opposed to a runtime error), `error` will consist of multiple lines including the filename and the exact line of code. This throws results way off.

For instance try the prior version with the following program:
```py
for i in range(10):
    print(i)
        print(i+1)
```